### ISSUE TITLE
Please support building with pre-C++11 STL

### ISSUE THREAD

**Issue by jeremyhu** [libproxy/libproxy#29]
https://github.com/libproxy/libproxy/commit/28620c7aeb3d1b54c83caf84778df8e095490820 seems to have inadvertently added a dependency on C++11 to libproxy by its use of std::map::at.  When building against older STL, libproxy fails to build:

<pre>
/opt/local/var/macports/build/_Users_jeremy_src_macports_trunk_dports_net_libproxy/libproxy/work/libproxy-0.4.13/libproxy/modules/config_kde.cpp:165:30: error: 
      no member named 'at' in 'std::map<std::basic_string<char>, std::basic_string<char>, std::less<std::basic_string<char> >,
      std::allocator<std::pair<const std::basic_string<char>, std::basic_string<char> > > >'
                return cache.at(key);
                       ~~~~~ ^
1 error generated.
</pre>

If you intend to require C++11, please document it.  If not, please provide an alternative implementation for this check in config_kde.cpp.

This was encountered by a MacPorts user.  See https://trac.macports.org/ticket/51309

**Comment by jeremyhu** [comment]
@Vogtinator I think we'll just revert 28620c7aeb3d1b54c83caf84778df8e095490820 on MacPorts until this can be addressed, so no rush.

**Comment by Vogtinator** [comment]
I was actually not aware that std::map::at is C++ >= 11 only. I'll fix that soon.

Regarding the issue, why do you compile the KDE module at all?

**Comment by DimStar77** [comment]
This has been fixed byr#30
